### ISSUE TITLE
DNS client: Don't try to use forwarder that is down

### ISSUE THREAD

**Issue by Revertron** [Revertron/Alfis#303]
Sometimes one of the forwarders is down, but ALFIS tries to use it for resolving, and gets timeouts.
We need to think of some algorithm that will use forwarders that are online, but check offline forwarders once in some time.

**Comment by FliKites** [comment]
Good idea. Have you thought of anything that might be possible with this?

**Comment by Revertron** [comment]
Not really. Didn't have an opportunity to think :)
